core/txpool/celo_validation.go

```diff
@@ -56,5 +61,16 @@ func CeloValidateTransaction(tx *types.Transaction, head *types.Header,
 		return exchange.ErrUnregisteredFeeCurrency
 	}
 
+	celoGasPrice, err := exchange.ConvertCurrencyToCelo(
+		currencyCtx.ExchangeRates,
+		tx.FeeCurrency(),
+		tx.GasFeeCap(),
+	)
+	if err != nil {
+		return err
+	}
+	if new(big.Int).SetUint64(opts.Config.Celo.EIP1559BaseFeeFloor).Cmp(celoGasPrice) == 1 {
```

Comment:
This seems to cause problems in tests, probably`Config.Celo` is `nil` in some cases.
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xf98521]

goroutine 5781 [running]:
github.com/ethereum/go-ethereum/core/txpool.CeloValidateTransaction(0xc001820690, 0xc0010af180?, {0x1f2b140?, 0xc000480380?}, 0xc0010af[190](https://github.com/celo-org/op-geth/actions/runs/12188965910/job/34003151772?pr=292#step:6:191), {0x0?, 0x0?})
	/runner/_work/op-geth/op-geth/core/txpool/celo_validation.go:72 +0x1a1
github.com/ethereum/go-ethereum/core/txpool/legacypool.(*LegacyPool).validateTxBasics(0xc0002d1800, 0xda12bf9a14d4a980?, 0x0)
	/runner/_work/op-geth/op-geth/core/txpool/legacypool/legacypool.go:667 +0x170
github.com/ethereum/go-ethereum/core/txpool/legacypool.(*LegacyPool).Add(0xc0002d1800, {0xc00040fc00, 0x80, 0x41e485?}, 0xa0?, 0x0)
[...]
```

Reply:
sorry, I forgot to mark the PR as draft initially